plugins/Diagnostics/Controller.php

```diff
@@ -35,6 +35,9 @@ public function configfile()
         $configValues = $this->configReader->getConfigValuesFromFiles();
         $configValues = $this->configReader->addConfigValuesFromSystemSettings($configValues, $allSettings);
         $configValues = $this->sortConfigValues($configValues);
+        $configValues = array_filter($configValues, function ($configSection) {
+            return !empty($configSection);
+        });
```

Comment:
this should be functionally the same
```suggestion
        $configValues = array_filter($configValues);
```